samples/nhibernate/simple/NHibernate_7/Client/App.config

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <configSections>
+    <section name="UnicastBusConfig"
+             type="NServiceBus.Config.UnicastBusConfig, NServiceBus.Core"/>
+  </configSections>
+
+  <connectionStrings>
+    <add name="NServiceBus/Persistence"
+         connectionString="Data Source=.;Database=Samples.NHibernate;Integrated Security=True"
+         providerName="System.Data.SqlClient" />
+  </connectionStrings>
+  
+  <UnicastBusConfig>
+    <MessageEndpointMappings>
```

Comment:
I think that if it is not a snippet rendered in the sample landing page, we should keep it aligned with other NSB 6 content which tries to consistently use the code-first routing to give users a clear sign that we're moving away from MEMs. @weralabaj what do you think?